### DISCUSSION TITLE
Поддержка АС

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#688] [General]
Привет! Поддержка АС еще актуально, а то последнее обновление было 22 октября?

**Comment by trickerer** [comment]
Обновлений не было из-за проблем с совместимостью. Новую систему гридов всё никак не могли пофиксить, уже вроде откатили.

`Если бы поддержка прекратилась соответствующий репозиторий и файлы тоже были бы удалены`
